### DISCUSSION TITLE
model regresi yang sudah di define dianggap not found

### DISCUSSION THREAD

**Discussion by AGelfP** [teamalgoritma/community#582] [Q&A] [answered]
Saya sudah memastikan bahwa model regresi yang dibuat sudah di define (namanya model_all). Terlampir berikut screenshot yang menunjukkan bahwa model tersebut sudah bisa dipanggil:
![image](https://user-images.githubusercontent.com/122977451/222622160-a439d234-b03b-40c3-8fef-673dce011fe1.png)

Tapi ketika mau dilakukan step-wise regression, justru dianggap tidak ada. Sudah mencoba clear up environment agar tidak tercampur dengan object-object dari project R lain juga masih tidak ketemu:
![image](https://user-images.githubusercontent.com/122977451/222622244-dc20522a-bddb-4b9d-813c-983430737736.png)

Kira-kira kenapa ya? Mohon bantuannya, terima kasih

**Comment by idoali** [comment, marked as answer]
Terima kasih atas pertanyaannya

Ketika membuat `model_all` bisa dipastikan lagi data yang dimasukkan benar pak. Karena sepertinya ketika bapak membuat `model_all` data yang dimasukkan tidak tertulis dengan benar (pada fungsi `lm` parameter `data =` yang bapak masukkan adalah `.` bukan nama data).
